pkg/util/time_test.go

```diff
@@ -174,3 +175,32 @@ func TestNewDisableableTicker_Disabled(t *testing.T) {
 		break
 	}
 }
+
+func TestUnixSecondsJSON(t *testing.T) {
+	now := time.Now()
+
+	type obj struct {
+		Val UnixSeconds `json:"val,omitempty"`
+	}
+
+	type testCase struct {
+		obj  obj
+		json string
+	}
+
+	for _, tc := range []testCase{
+		{obj: obj{Val: 0}, json: `{}`},
+		{obj: obj{Val: UnixSecondsFromTime(now)}, json: fmt.Sprintf(`{"val":%d}`, now.Unix())},
+	} {
+		t.Run(tc.json, func(t *testing.T) {
+			out, err := json.Marshal(tc.obj)
+			require.NoError(t, err)
+			require.Equal(t, tc.json, string(out))
```

Comment:
there's require.JSONEq which is less flaky

Reply:
Thanks, that's useful.